src/countItems.js

```diff
@@ -12,12 +12,21 @@
  *  string in the input array
  */
 function countItems(array) {
-  // This is your job. :)
-}
+  let dict = {}
 
+  for (let i = 0; i < array.length; i++) {
+    dict[array[i]]=null;
```

Comment:
This works because when you add a number to `null`, JS silently converts `null` to `0`. There's no reason to rely on that behavior here, though. Instead, initialize the dictionary values to `0`:

```javascript
dict[array[i]] = 0;
```